SwiftUIHW/SwiftUIHW/Services/ImageService.swift

```diff
@@ -0,0 +1,36 @@
+// ImageService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Сервис получения изображений
+final class ImageService: ObservableObject {
+    // MARK: - Public methods
+
+    func fetchImage(_ url: String) -> some View {
+        AsyncImage(url: URL(string: url)) { phase in
```

Comment:
```suggestion
        AsyncImage(url: URL(string: url)) { phase in
```
AsyncImage должен быть во вью а урл viewModel.ulr